examples/gcp_cli.py

```diff
@@ -16,7 +16,9 @@
 
 import argparse
 import json
-from libcloudforensics import gcp
+from libcloudforensics.providers.gcp.internal.project import GoogleCloudProject
```

Comment:
Please import the module and not the class directly:

```python
from libcloudforensics.providers.gcp.internal import project
project.GoogleCloudProject(...)
```

Alternatively, you can also define in /gcp/internal/__init__.py which modules to expose here, like

```python
from gcp import GoogleCloudProject # You're effectively importing a class and not a module, but it's really clear where it's coming from since it's all contained in the __init__ file, so I guess it's OK
```

So you can do

```python
from libcloudforensics.providers.gcp import internal as gcp_internal
gcp_internal.GoogleCloudProject
```

Reply:
I opt for the second approach, given the amount of submodules we have under `internal` I find it cleaner.